swap_meet/vendor.py

```diff
@@ -1,2 +1,79 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
+    
+    def __init__(self, inventory=[], items=[]):
+        self.inventory = inventory
+        self.items = items 
+        
+    def add(self, item_to_add):
+        self.inventory.append(item_to_add)
+        return item_to_add
+    
+    def remove(self, item_to_remove):
+        if item_to_remove in  self.inventory:
+            self.inventory.remove(item_to_remove)
```

Comment:
A very nitpicky point: the description says that if we find a matching object that we should return the object that was removed. `remove` looks for an item to match using `==` checks (more or less), so the item it finds in the list will _look like_ the query item, but might be a different instance. To "return the object that was  removed" we should find a matching item, pull it out of the list, and return _that_ instance.

Again, I'm super splitting hairs here! 😄